README.md

```diff
@@ -17,15 +18,10 @@
 
 ## Usage
 
-> Some usage instructions
+>  From the root directory of this service, grab dependencies by running npm install. Then run npm start to get the server going on localhost:3000. Then run npm run react-dev to transpose the es6 files into a bundle using webpack. Start your mongodb by typing in mongod into the console. Next, npm run seed-db. This script will create a properties database with 3 tables (properties comparablehomes and localhomes), and then populate the three tables with 100 documents each. 

```

Comment:
Can move the dependency text to the section down below

Reply:
@leightonchen made this change. Thanks!